include/rootfs.mk

```diff
@@ -60,6 +60,12 @@ ifdef CONFIG_CLEAN_IPKG
   endef
 endif
 
+# Remove not needed information from the control file
+# 1: roofs base path
+define clean_control_file
+	-sed -i '/^Maintainer: /d' $(1)/usr/lib/opkg/info/*.control
```

Comment:
Please be aware that we eventually plan your switch to APK which renders those changes obsolete. Just a comment

Reply:
Thanks for the hint.
If this is the case then I will discard the newly opened PR this commit, that is removing this Information from the control file. What information is stored on the rootfs target when using APK? Is there any information on this yet?